### ISSUE TITLE
`require-throws` ignores class constructors

### ISSUE THREAD

**Issue by FunctionDJ** [gajus/eslint-plugin-jsdoc#1692]
## Expected behavior

```ts
class MyClass {
	// below JSdoc has "Missing JSDoc @throws declaration."
	/**
	 *
	 */
	constructor() {
		throw new Error("hi")
	}
}
```

## Actual behavior

```ts
class MyClass {
	// no lint error for this JSdoc
	/**
	 *
	 */
	constructor() {
		throw new Error("hi")
	}
}
```

## ESLint Config

```js
export default defineConfig([
  jsdoc({
    config: "flat/recommended-typescript",
  }),
  {
    rules: {
      "jsdoc/require-throws": "error"
    }
  }
])
```

## Environment

- Node version: v25.6.1
- ESLint version v10.4.0
- `eslint-plugin-jsdoc` version: ^62.9.0
